libs/game/tilemap.ts

```diff
@@ -501,6 +501,21 @@ namespace tiles {
         return scene.tileMap.getTileImage(tiles.getTileLocation(col, row).tileSet);
     }
 
+    /**
+     * Returns true if the tile at the given location is the same as the given tile;
+     * otherwise returns false
+     * @param location
+     * @param tile
+     */
+    //% blockId=maplocationistile block="tile at $location is $tile"
+    //% location.shadow=mapgettile
+    //% tile.shadow=tileset_tile_picker tile.decompileIndirectFixedInstances=true
+    //% blockNamespace="scene" group="Tiles" blockGap=8
```

Comment:
Should this group be "collisions"  instead of "Tiles"?

Reply:
I don't think this block belongs in collisions, as this is just checking a location / does not involve a sprite 'hitting' a spot like the collisions do. Maybe the overlaps / hits wall blocks should, but I think we had decided on them being at the top of the tiles category so that they're more obvious / visible

Reply:
![image](https://user-images.githubusercontent.com/6107272/86822104-0acf0200-c040-11ea-8da5-6f562215dfe5.png)

We have above in the collisions?

Reply:
I suppose, I think of that one as `tile sprite is hitting on the [left]` which makes sense under the collision category.

Maybe we should rename Collisions to something else, the split between tiles and it feels fairly arbitrary to me